### PR TITLE
Added scenario [SORDEV-5967] "Add evidence fields for event clusters"…

### PR DESCRIPTION
**Pull request by mk-sgent**

### CODE REVIEW

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EditEventPage.java [hunk 1]
```diff
@@ -74,6 +74,60 @@ public class EditEventPage {
...
+  public static final By PRIMARY_MODE_OF_TRANSMISSION_COMBOBOX =
+      By.cssSelector("[id='diseaseTransmissionMode'] [class='v-filterselect-button']");
+  public static final By EPIDEMIOLOGICAL_EVIDENCE_OPTIONS =
+      By.cssSelector("#epidemiologicalEvidence .v-select-option");
+  public static final By LABORATORY_DIAGNOSTIC_EVIDENCE_OPTIONS =
+      By.cssSelector("#laboratoryDiagnosticEvidence .v-select-option");
+  public static final By STUDY_EPIDEMIOLOGICAL_EVIDENCE_BUTTON =
+      By.xpath("//label[text()='Studie']");
+  public static final By CASE_CONTROL_STUDY_EPIDEMIOLOGICAL_EVIDENCE_BUTTON =
+      By.xpath("//label[text()='Fall-Kontroll-Studie']");
```
Comment (jp-sgent) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EditEventPage.java:87]: I suggest adding a "_DE" suffix to the names of german-specific variables.

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java [hunk 2]
```diff
@@ -154,6 +179,95 @@ public EditEventSteps(
...
+        });
+
+    When(
+        "I click on Epidemiological evidence with ([^\"]*) option",
+        (String option) -> {
+          webDriverHelpers.clickWebElementByText(EPIDEMIOLOGICAL_EVIDENCE_OPTIONS, option);
+        });
+
+    When(
+        "I thick all options for Study on Epidemiological evidence for De version",
```
Comment (jp-sgent) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java:201]: If by "thick" you mean marking a checkbox, then it's a typo (should be "tick", not "thick").

**Review by jp-sgent — APPROVED** [review]

**Comment by mk-sgent** [comment]
SORQA-185

Jenkins jobs:
**First**
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/735/allure/
**Second**
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/736/allure/

**Review by pk-sgent — APPROVED** [review]

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java [hunk 3]
```diff
@@ -154,6 +179,94 @@ public EditEventSteps(
...
         });
 
+    When(
+        "I check CLUSTER option on edit Event page",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(EVENT_CLUSTER_EDIT);
+        });
+
+    When(
+        "I select {} option from Primary Mode Of Transmission Combobox on edit Event page",
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java:189]: should {string}

File: sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature [hunk 4]
```diff
@@ -453,4 +453,26 @@ Feature: Create events
...
+    And I tick the all options for Study on Epidemiological evidence for De version
+    And I tick the all options for Explorative survey of affected people on Epidemiological evidence for De version
+    And I tick the all options for Descriptive analysis of ascertained data on Epidemiological evidence for De version
+    And I tick the all options for Suspicion on Epidemiological evidence for De version
+    Then I click on Laboratory diagnostic evidence with UNBEKANNT option
+    And I click on Laboratory diagnostic evidence with NEIN option
+    And I click on Laboratory diagnostic evidence with JA option
+    And I tick the all options for Verification of at least two infected or diseased persons on Laboratory diagnostic evidence for De version
+    And I tick the all options for Verification on materials on Laboratory diagnostic evidence for De version
+    And I click on SAVE button in edit event form
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature:478]: test is not completed, we select all options here, but we don't also check the test case checks after.

image

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**